ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -175,6 +178,17 @@ class PhaserUnit extends PhaserAnimatedEntity {
 		}
 	}
 
+	private scaleElements (height): void {
+		const defaultZoom = ige.game.data.settings.camera?.zoom?.default || 1000;
+		const targetScale = height / defaultZoom;
```

Comment:
this scales the elements but does not keep a consistent margin between the attribute bars. try it out on my test game or add a second unit attribute bar and you will see it